Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -174,42 +174,68 @@
                         <NoteTimelineComponent :is-add-mode="true" />
                     </b-col>
                 </b-row>
-                <b-row class="view-selector justify-content-end">
-                    <b-col cols="auto" class="pr-0">
-                        <b-btn
-                            class="month-view-btn btn-outline-primary px-2 m-0"
-                            :class="{ active: !isListView }"
-                            @click.stop="toggleMonthView"
-                        >
-                            Month
-                        </b-btn>
-                    </b-col>
-                    <b-col cols="auto" class="pl-0">
-                        <b-btn
-                            class="list-view-btn btn-outline-primary px-2 m-0"
-                            :class="{ active: isListView }"
-                            @click.stop="toggleListView"
-                        >
-                            List
-                        </b-btn>
-                    </b-col>
-                </b-row>
                 <LinearTimeline
                     v-show="isListView && !isLoading"
                     :timeline-entries="timelineEntries"
                     :is-visible="isListView"
                     :total-entries="getTotalCount()"
                     :filter-text="filterText"
                     :filter-types="filterTypes"
-                />
+                >
+                    <b-row
```

Comment:
some repeated code...  We can live with it, but we should make it a component

Reply:
yeah, I have tried to move it to a new MonthListToggleComponent, but it doesn't seem to render a vue component in a slot.